3-4/GalilSup/Db/Makefile

```diff
@@ -11,6 +11,8 @@ include $(TOP)/configure/CONFIG
 # Create and install (or just install) into <top>/db
 # databases, templates, substitutions like this
 
+DB += $(patsubst ../%, %, $(wildcard ../*.template))
```

Comment:
Most modules seem to build templates into a top level db directory. Then in the IOC startup script or substitution file I can just refer to the file name rather than the path to the Db directory, eg:

file "galil_dmc_ctrl.template"
{
pattern
{   P,        PORT,    SCAN,       DEFAULT_HOMETYPE, DEFAULT_LIMITTYPE, PREC}
 {  "BL100:Mot:DMC1:", "DMC1", "Passive",  1,                1,                 5 }
}

If a template is changed then a new 'make' will just copy the changed file into the top level db directory.

If you don't want to add this part then that's fine. I'll just add it to our local copy since it's only a single line.